projects/ngx-prx-styleguide/src/lib/tz-datepicker/tz-timestamp-validator.directive.ts

```diff
@@ -0,0 +1,25 @@
+import { AbstractControl, Validator, NG_VALIDATORS, ValidatorFn, ValidationErrors, FormControl } from '@angular/forms';
+import { Directive, Input } from '@angular/core';
+import * as momentNs from 'moment';
+import 'moment-timezone/moment-timezone';
+const moment = momentNs;
+
+@Directive({
+  selector: '[prxTzTimestamp]',
+  providers: [{ provide: NG_VALIDATORS, useExisting: TzTimestampValidatorDirective, multi: true }]
+})
+export class TzTimestampValidatorDirective implements Validator {
+  @Input('prxTzTimestamp') timestampFormat: string; // 12-hour (hh:mm:ss) or 24-hour (HH:mm:ss)
```

Comment:
should add a tslint ignore to this line to rename the format input
`/* tslint:disable-next-line:no-input-rename */`